src/lib/common/sol-mainloop-impl-riot.c

```diff
@@ -238,16 +87,16 @@ timex_set_until_next_timeout(timex_t *timex)
 void
 sol_mainloop_impl_run(void)
```

Comment:
This should be the `sol_mainloop_run()` function, which should call `sol_mainloop_impl_iter()` (or other similarly-named function), left to be implemented by the platform-specific code. This way, there's no need to expose the `_common_` functions.